Client/RecommendationRequestClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Client;
+
+interface RecommendationRequestClient
+{
+    /**
+     * @param int $userId
```

Comment:
Meaningful description here as well ;-)

What about `Returns $limit recommendations for a $locationId and a $userId based on a $scenarioId.`.

I often find that writing a sentence that uses the method's and arguments' names helps clarifying what it does.